src/student_success_tool/modeling/evaluation.py

```diff
@@ -644,6 +644,59 @@ def plot_shap_beeswarm(shap_values):
     return plt.gcf()
 
 
+def fnpr_group_plot(fnpr_data: list) -> matplotlib.figure.Figure:
```

Comment:
for consistency with my recent renaming:

```suggestion
def plot_group_fnpr(fnpr_data: list) -> matplotlib.figure.Figure:
```